src/main/java/org/hashids/Hashids.java

```diff
@@ -383,6 +402,6 @@ private static Long unhash(String input, String alphabet) {
    * @return Hashids algorithm version implemented.
    */
   public String getVersion() {
-    return "1.0.0";
+    return "1.0.4-SNAPSHOT";
```

Comment:
Yes, but there must be a way to view that there are new releases (with bug fixes, small improvements, etc. - without changing the algorithm). How can you do this without changing the version? 

If it is so important to you to respect the JS version, let's add a new number,  in the end.

Reply:
You don't check version using a method, you have dependencies managers, like maven, gradle, ivy, etc. This is plain wrong do it in code, also a pain to update a method every release.

You have to think out-of-the-box here, this is a large project with several implementations across the board, you have to stick to the same conventions, otherwise it became a mess.

Reply:
Yes, it is not advisable to check the version in this way but I'm not the one that I've created this method. I've just updated it.

Reply:
I agree, but you've said "Yes, but there must be a way to view that there are new releases". This is not the purpose of this method(The javadoc is pretty clear to me).

Reply:
I've said that because I've thought that someone is using your method and the version is not the correct one (e.g. the current one is [1.0.3](https://maven-badges.herokuapp.com/maven-central/org.hashids/hashids)).